releases/latest/kernel-slim/Dockerfile.ubi.openjdk11

```diff
@@ -1,15 +1,40 @@
-FROM icr.io/appcafe/ibm-semeru-runtimes:open-11-jdk-ubi
+FROM icr.io/appcafe/ibm-semeru-runtimes:open-11-jdk-ubi AS getRuntime
 
 USER root
 
-ARG LIBERTY_VERSION=23.0.0.5
-ARG LIBERTY_SHA=64fdc2ab454e81337631bc96a70d50cb710aa8ac
-ARG LIBERTY_BUILD_LABEL=cl230520230514-1901
+ARG LIBERTY_VERSION=23.0.0.6
+ARG LIBERTY_SHA=8b2e46dcf1f14112e901d33e3f92a0b4e0177197
 ARG LIBERTY_DOWNLOAD_URL=https://repo1.maven.org/maven2/io/openliberty/openliberty-kernel/$LIBERTY_VERSION/openliberty-kernel-$LIBERTY_VERSION.zip
+
+ARG VERBOSE=false
+
+# If there is a local copy of the image use that instead
+COPY resources/ /tmp/
+
+# Install Open Liberty
+RUN yum -y install shadow-utils wget unzip openssl \
```

Comment:
openssl is needed at runtime to import the mounted certificates. Since only the wlp runtime is copied from this stage to the next stage, I believe this package won't be included in the final image

Reply:
For consistency, I've explicitly added openssl to all the images